Range check makes the release fail

https://github.com/quarkusio/quarkus/pull/11259/files doesn't work when doing the release.

I will revert it for now in the 1.8 branch.

```
[ERROR]   AddExtensionMojoTest.testAddMultipleDependency:65 » MojoExecution Failed to in...
[ERROR]   AddExtensionMojoTest.testAddSingleDependency:51 » MojoExecution Failed to init...
[ERROR]   AddExtensionsMojoTest>AddExtensionMojoTest.testAddMultipleDependency:65 » MojoExecution
[ERROR]   AddExtensionsMojoTest>AddExtensionMojoTest.testAddSingleDependency:51 » MojoExecution
```

```
Caused by: java.lang.IllegalStateException: Failed to load the bundled platform artifacts
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.resolveJsonArtifactFromBom(QuarkusJsonPlatformDescriptorResolver.java:367)
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.resolveJsonDescriptor(QuarkusJsonPlatformDescriptorResolver.java:233)
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.resolve(QuarkusJsonPlatformDescriptorResolver.java:158)
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.resolveLatestFromBom(QuarkusJsonPlatformDescriptorResolver.java:128)
    at io.quarkus.maven.CreateUtils.resolvePlatformDescriptor(CreateUtils.java:101)
    at io.quarkus.maven.QuarkusProjectMojoBase.resolvePlatformDescriptor(QuarkusProjectMojoBase.java:112)
    ... 68 more
Caused by: io.quarkus.platform.descriptor.resolver.json.VersionNotAvailableException: Failed to resolve the latest version of io.quarkus:quarkus-bom::pom:[1.8-snapshot, 1.9)
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.loadFromBomCoords(QuarkusJsonPlatformDescriptorResolver.java:387)
    at io.quarkus.platform.descriptor.resolver.json.QuarkusJsonPlatformDescriptorResolver.resolveJsonArtifactFromBom(QuarkusJsonPlatformDescriptorResolver.java:364)
    ... 73 more
```

--- a/devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java
+++ b/devtools/maven/src/main/java/io/quarkus/maven/CreateUtils.java
@@ -85,9 +85,8 @@ public final class CreateUtils {
                 DefaultArtifactVersion pluginVersion = new DefaultArtifactVersion(baseVersion);
                 int majorVer = pluginVersion.getMajorVersion();
                 int minorVer = pluginVersion.getMinorVersion();
-                version = "[" + majorVer + "." + minorVer + "-snapshot, " + majorVer + "." + (minorVer + 1) + ")";
+                version = "[" + majorVer + "." + minorVer + "-alpha, " + majorVer + "." + (minorVer + 1) + ")";
             }
-
         }
 
         final QuarkusPlatformDescriptor platform;

